thewalrus/quantum.py

```diff
@@ -713,3 +716,68 @@ def is_classical_cov(cov, hbar=2, atol=1e-08):
         if np.all(vals >= 0):
             return True
     return False
+
+
+def gen_single_mode_dist(s, nmax=50, N=1):
+    """Generate the photon number distribution of N identical single mode squeezed states with squeezing parameter s up to cutoff nmax
+    Args:
+        s (float): squeezing paramer
+        nmax (int): Fock cutoff
+        N (float): Number of squeezed state(s).
+    Returns:

```

Comment:
Don't forget to include linebreaks in the docstring!
```suggestion

    Returns:
```